src/status.rs

```diff
@@ -184,12 +183,6 @@ impl StatusCode {
     }
 }
 
-impl fmt::Debug for StatusCode {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        fmt::Debug::fmt(&self.0, f)
```

Comment:
This impl was preferred because seeing `status: 200` is cleaner than `status: StatusCode(200)`.